packages/server/graphql/mutations/helpers/createTeamAndLeader.ts

```diff
@@ -74,7 +74,8 @@ export default async function createTeamAndLeader(userId: string, newTeam: Valid
         type: CREATED_TEAM,
         userId,
         teamId,
-        orgId
+        orgId,
+        isActive: true
```

Comment:
good find. Would you mind creating a new type for this similar to https://github.com/ParabolInc/parabol/blob/master/packages/server/database/types/TimelineEventJoinedParabol.ts#L12? that way if we make any other chances to the class we won't have this bug come up again.

Reply:
Yeah sounds good, added.